requirements.txt

```diff
@@ -3,7 +3,7 @@ cycler==0.10.0
 kiwisolver==1.1.0
 matplotlib==3.1.3
 numpy==1.18.1
-opencv-python==3.3.1
+opencv-python==3.4.2.17
 pandas==1.0.2
 pyparsing==2.4.6
 python-dateutil==2.8.1
```

Comment:
scikit-image is one of the dependencies. A tested list of requirements is here (https://github.com/czbiohub/pysero/blob/master/docs/installation.md).